test/integration/python_system_loader.cc

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <gz/transport/Node.hh>
+
+#include "gz/sim/Server.hh"
+#include "gz/sim/Util.hh"
+#include "gz/sim/components/Model.hh"
+#include "gz/sim/components/Name.hh"
+#include "helpers/EnvTestFixture.hh"
+#include "plugins/MockSystem.hh"
+
+using namespace gz;
+namespace components = gz::sim::components;
+
+class PythonSystemLoaderTest : public InternalFixture<::testing::Test>
+{
+};
+
+/////////////////////////////////////////////////
+void worldReset()
+{
+  gz::msgs::WorldControl req;
```

Comment:
include message: gz::msgs::WorldControl and gz::msgs::Boolean

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2081/commits/f30128ff50d9453168ec88e99e3d1b41126a7bc4